airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/split_secrets/SecretsHelpers.java

```diff
@@ -178,6 +178,7 @@ public static List<String> getSortedSecretPaths(final JsonNode spec) {
             .anyMatch(field -> field.getKey().equals(JsonSecretsProcessor.AIRBYTE_SECRET_FIELD)))
         .stream()
         .map(JsonPaths::mapJsonSchemaPathToJsonPath)
+        .distinct()
```

Comment:
can you add a unit test for this?

Reply:
added unit tests. will attach screenshots with failed tests without this change

Reply:
can you add in the javadoc comment that it will not return duplicates?